_data/classes.yml

```diff
@@ -62,7 +62,7 @@
     - "explore the ethics behind applications of AI and ML"
   path: ml
   lead:
-    name: "Arjun"
+    name: "Ava"
     hash: asubramonian
```

Comment:
Please change the hash from aubramonian to aasmani as well. The hash is based off of the person first name and last name (ie Ava's last name is asmani so her hash is aasmani). By changing the hash, this will change where the link directs to c: